src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1703,6 +1703,16 @@ def update_app_service_plan(instance, sku=None, number_of_workers=None):
     return instance
 
 
+def show_plan(cmd, resource_group_name, name):
+    client = web_client_factory(cmd.cli_ctx)
+    serverfarm_url_base = 'subscriptions/{}/resourceGroups/{}/providers/Microsoft.Web/serverfarms/{}?api-version={}'
+    subscription_id = client.app_service_plans._config.subscription_id
+    serverfarm_url = serverfarm_url_base.format(subscription_id, resource_group_name, name, client.DEFAULT_API_VERSION)
+    request_url = client.app_service_plans._client._base_url + serverfarm_url
+    response = send_raw_request(cmd.cli_ctx, "GET", request_url)
```

Comment:
Could we use `AppServicePlansOperations.get()` [code linke](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appservice/azure-mgmt-web/azure/mgmt/web/v2020_09_01/operations/_app_service_plans_operations.py#L201) the python SDK instead of requesting REST directly

Reply:
`AppServicePlansOperations.get()` API is throwing exception as a HTML response, but we are not caching this since CLI expects JSON response. To catch the HTML response we should make a HTTP call explicitly. So not possible to make a call from SDK directly.

Reply:
Got it, thanks~ The standard of REST API should be JSON response. May I ask why does this REST return HTML?